src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -7,6 +7,9 @@
 import mate.academy.hibernate.relations.service.ActorService;
 import mate.academy.hibernate.relations.service.CountryService;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.service.impl.ActorServiceImpl;
```

Comment:
Ensure that the `ActorServiceImpl` class is correctly implemented to handle session management. Consider using try-with-resources for managing the `Session` to ensure it is always closed properly, even if exceptions occur.